### PR TITLE
Fix issue #70

### PR DESCRIPTION
**Pull request by sajithaliyanage**
- Closes https://github.com/cibr-qcri/kansa/issues/70

### CODE REVIEW

File: api/utils/contracts.js [hunk 1]
```diff
@@ -20,7 +25,7 @@ exports.transformContractInfo = (row) => {
     totalSpentUSD: row.total_spent_usd,
     totalReceived: toEther(row.total_received) + ' Ether',
     totalReceivedUSD: row.total_received_usd,
-    securityScore: row.security_score,
+    securityScore: securityScore,
```
Comment (boshmaf) [inline api/utils/contracts.js:28]: Please use JS shorthand for key/value pairs (if key name is equal to value name, just pass one). So in this example, 
```
{
   ...
    totalReceivedUSD: row.total_received_usd,
    securityScore,
    ...
}
```

File: api/utils/contracts.js [hunk 2]
```diff
@@ -93,6 +98,32 @@ exports.getEdgeWidth = (value) => {
...
+  } else if (value > 0.25) {
+    securityScore += ' (Medium)';
+  } else {
+    securityScore += ' (Low)';
+  }
+
+  return securityScore;
+};
+
+exports.getContractStandard = (is_erc20, is_erc721) => {
```
Comment (boshmaf) [inline api/utils/contracts.js:114]: Better **pass the row** here, and then check row.is_erc20, etc. Also, this it is better to **rename this function to getTokenStandard** not contract. Everything is about the contract (the file is called contract.js after all).

File: api/utils/contracts.js [hunk 3]
```diff
@@ -11,6 +11,11 @@ exports.transformContractInfo = (row) => {
     tokenStandard = 'Other';
```
Comment (boshmaf) [inline api/utils/contracts.js:11]: Use `const tokenStandard  = getTokenStandard(row)` as the function is a contract util function now.

File: api/utils/contracts.js [hunk 4]
```diff
@@ -11,6 +11,11 @@ exports.transformContractInfo = (row) => {
...
   }
 
+  let securityScore = row.security_score;
+  if (securityScore != -1) {
+    securityScore = this.getSecurityScore(securityScore);
+  }
```
Comment (boshmaf) [inline api/utils/contracts.js:17]: Here, you're not getting the security score, you're annotating or labelling the security score as a string. The function name is confusing. Please use proper function names (see suggestion below when you define it).

File: api/utils/contracts.js [hunk 5]
```diff
@@ -93,6 +98,32 @@ exports.getEdgeWidth = (value) => {
   }
 };
 
+exports.getSecurityScore = (value) => {
```
Comment (boshmaf) [inline api/utils/contracts.js:101]: Rename to something that describes what you're doing. You're not getting the security score (which is a float number), you're annotating it with some string and returning a string instead. So maybe **annotateSecurityScore** is a better name, as you're explaining what the float number means).

**Review by boshmaf — CHANGES_REQUESTED** [review]

File: api/utils/contracts.js (hunk 4 above)
Comment (sajithaliyanage) [reply api/utils/contracts.js]: Fixed

File: api/utils/contracts.js (hunk 5 above)
Comment (sajithaliyanage) [reply api/utils/contracts.js]: Fixed

File: api/utils/contracts.js (hunk 3 above)
Comment (sajithaliyanage) [reply api/utils/contracts.js]: Fixed

File: api/utils/contracts.js (hunk 2 above)
Comment (sajithaliyanage) [reply api/utils/contracts.js]: Fixed

**Review by boshmaf — APPROVED** [review]

**Pull request merged**